horace_core/sqw/axes_blocks/@AxesBlockBase/private/calc_bin_volume_.m

```diff
@@ -0,0 +1,34 @@
+function volume = calc_bin_volume_(~,ax_in_cell)
```

Comment:
If this doesn't take `obj` should be static

Reply:
spherical projection overload do take object